surfer/viz.py

```diff
@@ -2751,6 +2753,16 @@ def _scale_sequential_lut(lut_table, fmin, fmid, fmax):
     return lut_table_new
 
 
+def _check_limits(fmin, fmid, fmax, extra='f'):
+    """Check for monotonicity."""
+    if fmin >= fmid:
+        raise ValueError('%smin must be < %smid, got %s >= %s'
```

Comment:
`%s` with floats is going to make this a message with a lot of decimal values, right? Maybe add a formatting code to improve signal to noise?